README.md

```diff
@@ -12,33 +12,100 @@ They can also be used independently of Witan as a standalone demographic models
 
 Current status:
 * Population projections: First release of a minimal version coming soon!
-* Small area projections: Not started. Waiting to finish the population projections.
 
-See the [docs](doc/intro.md) for more info!
+See the [docs](doc/intro.md) for more info about the methodology.
+
+## Requirements
+
+You need to have [Leiningen](http://leiningen.org/) installed to run the projections.
 
 ## Installation
 
-Download from http://example.com/FIXME.
+1) Download or clone this repository.
+
+2) Use the terminal to navigate to the directory that the repo is in, for example:
+```
+	$ cd witan.models.demography
+```
+
+3) Create an executable .jar file:
+```
+	$ lein clean
+
+	$ lein uberjar
+```
+
+Copy the filepath to the standalone file, starting at the `target` directory, to use later. For example:
+```
+    target/uberjar/witan.models.demography-0.1.0-SNAPSHOT-standalone.jar
+```
 
 ## Usage
+Population projections can be produced by running the executable .jar file from the command line:  
 
-FIXME: explanation
+```
+    $ java -jar [path/to/jar] -c ["GSS code"] -i [optional/path/to/input/config/file] -o [optional/path/to/output/csv] 
+```
 
-    $ java -jar witan.models.demography-0.1.0-standalone.jar [args]
+For example:
+```
+    $ java -jar target/uberjar/witan.models.demography-0.1.0-SNAPSHOT-standalone.jar -c "E06000023" -i "default_config.edn" -o "ccm_projections.csv" 
+```
+A GSS code must always be specified to tell the model which geographic area the proejctions are for. See the GSS codes that are accepted [here](#gss-code).
 
 ## Options
 
-FIXME: listing of options this app accepts.
+The projections can be customised by adjusting the options below. Read more about how the .jar file works and what input data is expected in the [docs](doc/run-model.md).
+
+* [`-c` or `--gss-code`](#gss-code)
+* [`-i` or `--input-config`](#input-config)
+* [`-o` or `--output-projections`](#output-projections)
+
+
+The last two options have default values if not specified:
+
+* default input configuration filepath: "default_config.edn"
+* default output projection filepath: "ccm_projections.csv"
+
```

Comment:
Maybe add something like:
These default settings mean the "default_config.edn" file is located inside this repository and the output will be created as "ccm_projections.csv" inside this repository as well.